lib/doh.c

```diff
@@ -969,6 +1313,20 @@ CURLcode Curl_doh_is_resolved(struct Curl_easy *data,
     } /* address processing done */
 
     /* Now process any build-specific attributes retrieved from DNS */
+#ifdef USE_HTTPSRR
+    if(de.numhttps_rrs > 0 && result == CURLE_OK && *dnsp) {
+      int rv;
```

Comment:
```suggestion
      CURLcode result;
```

We try to use the variable name `result` for the generic local variable that holds return code from functions returning `CURLcode`.

Reply:
done